docs/configuration_options.md

```diff
@@ -0,0 +1,275 @@
+Should return an object with an `id` method, and optionally `username` and
+`email` methods.
+
+If not using Rails:
+
+Populate the Rack `person_data` key with a hash containing `:id`, and optionally
+`:username` and `:email`.
+
+### person_id_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's id (a string). Ignored if `person_method`
+present.
+
+### person_username_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's username. Ignored if `person_id_method` not
+present. Ignored if `person_method` present.
+
+### person_email_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's email address. Ignored if `person_id_method`
+not present. Ignored if `person_method` present.
+
+### populate_empty_backtraces
+
+If you report a `new` exception, but do not `raise` it, the backtraces in Rollbar
+will be empty. Set `populate_empty_backtraces` to `true` to have Rollbar load
+the traces before sending them.
+
+### request_timeout
+
+**Default** `3`
+
+Set the request timeout for sending POST data to Rollbar.
+
+### root
+
+Set the server root, all stack frames outside that root are considered
+'non-project' frames. Also used to setup Github linking.
+
+### safely
+
+**Default** `false`
+
+When `true` evaluates `custom_data_method` returns `{}` if an error,
+otherwise reports the error to Rollbar.
+
+### scrub_fields
+
+Fields to scrub out of the parsed request data. Will scrub from `GET`, `POST`,
+url, and several other locations. Does not currently recurse into the full
+payload.
+
+### scrub_user
+
+**Default** `true`
+
+Set to false to skip scrubbing user out of the URL.
+
+### scrub_password
+
+Set to false to skip scrubbing password out of the URL.
+
+### user_ip_obfuscator_secret
+
+A string used hash IP addresses when obfuscating them.
+
+### randomize_scrub_length
+
+**Default** `true`
+
+When true randomizes the number of asterisks used to display scrubbed fields.
+
+### uncaught_exception_level
+
+**Default** `error`
+
+Use this field to select a different level for uncaught errors (like `critical`,
+or `warning`).
+
+### scrub_headers
+
+**Default** `["Authentication"]`
+
+The headers to scrub.
+
+### sidekiq_threshold
+
+**Default** `0`
+
+The number of job re-tries before reporting an error to Rollbar via Sidekiq.
+Ignored unless you've called `use_sidekiq`.
+
+### verify_ssl_peer
+
+**Default** `true`
+
+By default we use `OpenSSL::SSL::VERIFY_PEER` for SSL. Although we don't
+recommend changing it, you can disable peer verification in case you experience
+SSL connection problems.
+
+### use_async
+
+**Default** `false`
+
+When `true` indicates you wish to send data to Rollbar asynchronously. If
+installed, uses `girl_friday`, otherwise defaults to `Threading`.
```

Comment:
If you use the ` characters you should use existing methods, constants, etc...

In ruby `Threading` is not a constant and doesn't exist in the core library, use `Thread` instead.